front/components/assistant/conversation/ConversationTitle.tsx

```diff
@@ -22,12 +22,14 @@ import { classNames } from "@app/lib/utils";
 
 export function ConversationTitle({
   owner,
+  conversationId,
```

Comment:
We can have a conversationId before we have a conversation (shallow page change before the swr hook is executed)